### PR TITLE
Fix open_orbit_dataset and ope_attitude_dataset 

### PR DESCRIPTION
**Pull request by aurghs**
Fix output format of open_orbit_dataset and ope_attitude_dataset. Add tests

### CODE REVIEW

File: xarray_sentinel/esa_safe.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import functools
 import os
 import pathlib
+import re
```
Comment (alexamici) [inline xarray_sentinel/esa_safe.py:4]: flake8 returns: `F401 're' imported but unused`

flake8 errors are now failures in the CI, maybe we need to make them so.

File: xarray_sentinel/sentinel1.py [hunk 2]
```diff
@@ -100,23 +101,17 @@ def open_orbit_dataset(product_path: str) -> xr.Dataset:
...
+
     for k in range(shape):
-        data_vars["time"].append(orbit[k]["time"])
-        data_vars["x"].append(orbit[k]["position"]["x"])
-        data_vars["y"].append(orbit[k]["position"]["y"])
-        data_vars["z"].append(orbit[k]["position"]["z"])
-        data_vars["x"].append(orbit[k]["velocity"]["x"])
-        data_vars["y"].append(orbit[k]["velocity"]["y"])
-        data_vars["z"].append(orbit[k]["velocity"]["z"])
+        data_vars["time"][1].append(orbit[k]["time"])
```
Comment (alexamici) [inline xarray_sentinel/sentinel1.py:108]: I `time` considered a coordinate? You add it to `data_vars` not `coords` and only test that it is a dimension, so I'm not sure.

File: xarray_sentinel/sentinel1.py [hunk 3]
```diff
@@ -100,23 +101,17 @@ def open_orbit_dataset(product_path: str) -> xr.Dataset:
...
+        data_vars["x"][1].append(orbit[k]["position"]["x"])
+        data_vars["y"][1].append(orbit[k]["position"]["y"])
+        data_vars["z"][1].append(orbit[k]["position"]["z"])
+        data_vars["vx"][1].append(orbit[k]["velocity"]["x"])
```
Comment (alexamici) [inline xarray_sentinel/sentinel1.py:112]: Minor: `vx` is short and pretty obvious, but less explicit than `velocity_x`. Not sure which of the two is best.

File: xarray_sentinel/sentinel1.py [hunk 4]
```diff
@@ -132,13 +127,16 @@ def open_orbit_dataset(product_path: str) -> xr.Dataset:
...
     return ds
 
 
-def open_root_dataset(filename: str) -> xr.Dataset:
-    manifest = esa_safe.open_manifest(filename)
+def open_root_dataset(product_path: str) -> xr.Dataset:
+    manifest = esa_safe.open_manifest(product_path)
     product_attrs, product_files = esa_safe.parse_manifest_sentinel1(manifest)
-    product_attrs["groups"] = ["orbit"] + product_attrs["xs:instrument_mode_swaths"]
+    product_attrs["groups"] = ["orbit", "attitude"] + product_attrs[
```
Comment (alexamici) [inline xarray_sentinel/sentinel1.py:137]: `group="gcp"` is also already implemneted, please add it.

**Review by alexamici — CHANGES_REQUESTED** [review]

File: xarray_sentinel/sentinel1.py (hunk 2 above)
Comment (aurghs) [reply xarray_sentinel/sentinel1.py:108]: It's not necessary to add coordinates explicitly. If a variable and its dimension have the same name, then for xarray it is a coordinate. But probably would be better to add it explicitly.

File: xarray_sentinel/esa_safe.py (hunk 1 above)
Comment (aurghs) [reply xarray_sentinel/esa_safe.py:4]: Thanks I fux it

File: xarray_sentinel/sentinel1.py (hunk 3 above)
Comment (aurghs) [reply xarray_sentinel/sentinel1.py:112]: `velocity_x` is too long for me. but I should add it in the long name

Comment (aurghs) [reply xarray_sentinel/sentinel1.py:117]: added in long name

**Pull request merged**